lib/services/runtime/handlers/reminder.ts

```diff
@@ -3,11 +3,16 @@ import { Node as BaseNode } from '@voiceflow/base-types';
 import { replaceVariables } from '@voiceflow/common';
 import { HandlerFactory } from '@voiceflow/general-runtime/build/runtime';
 import axios from 'axios';
-import moment from 'moment';
+import dayjs from 'dayjs';
+import dayjs_customParseFormat from 'dayjs/plugin/customParseFormat';
+import dayjs_utc from 'dayjs/plugin/utc';
 
 import { S, T } from '@/lib/constants';
 import { AlexaHandlerInput } from '@/lib/services/alexa/types';
 
+dayjs.extend(dayjs_utc);
+dayjs.extend(dayjs_customParseFormat);
```

Comment:
calling `.extend()` here will affect all usage of dayjs in this project right? side-effects like that should probably belong in a top-level file to make it clear that it's a global change

we seem to be using the very top of `start.ts` for this kind of thing:

https://github.com/voiceflow/alexa-runtime/blob/7751f9bf46fa94016bd37f951e48341e4b2d3f31/start.ts#L1-L2